src/net/JNetReflector/Program.cs

```diff
@@ -34,6 +34,8 @@ static void Main(string[] args)
 
                 JNetReflectedCore.CreateGlobalInstance();
 
+                ReflectionUtils.SetHandlerAndLevel((o1, o2) => Console.WriteLine(o2), JNetReflectedCore.TraceLevel);
```

Comment:
The data reported are printed into Console, anyway we can add the option to write into a file as an option. Add a new command-line switch to output to a file